templates/cli/lib/commands/command.js.twig

```diff
@@ -37,13 +40,27 @@ const {{ service.name | caseLower }}{{ method.name | caseUcfirst }} = async ({ {
     if (!fs.lstatSync(folderPath).isDirectory())
         throw new Error('The path is not a directory.');
     
+    const ignorer = ignore();
+
+    const xfunction = localConfig.getFunction(functionId);
```

Comment:
```suggestion
    const func = localConfig.getFunction(functionId);
```

In deploy.js we're using `func` so let's try to be consistent. 

We need to either change both occurances to `func` or `xfunction`

Can you check whats a more widely accepted / accurate convention ?